src/pages/workspaces/List.js

```diff
@@ -191,16 +182,15 @@ export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
           ]),
           viewToggleButtons(listView, listView => this.setState({ listView }))
         ]),
-        div({ style: styles.cardContainer }, [
+        div({ style: styles.cardContainer(listView) }, [
           h(NewWorkspaceCard, {
-            listView,
             onClick: () => this.setState({ creatingNewWorkspace: true })
           }),
-          _.map(workspace => {
-            return h(WorkspaceCard, { listView, workspace, key: workspace.workspace.workspaceId })
-          }, data),
-          !isDataLoaded && spinnerOverlay
-        ]),
+          listView ?
+            div({ style: { flex: 1, minWidth: 0, margin: '0.75rem 1rem 0rem 0.75rem' } },
+              renderedWorkspaces
```

Comment:
nit: This technically works because `renderedWorkspaces` is an array, but for clarity we should probably have an extra pair of brackets that serve as the 'children' collection.

Reply:
Should I put an extra pair of brackets on both instances of `renderedWorkspaces`?